src/Services/ConsignmentEncode.php

```diff
@@ -106,7 +107,7 @@ public static function encodeExtraOptions(array $consignmentEncoded, AbstractCon
         if ($consignment->getInsurance() > 1) {
             $consignmentEncoded['options']['insurance'] = [
                 'amount'   => (int) $consignment->getInsurance() * 100,
-                'currency' => 'EUR',
+                'currency' => ExportRow::CURRENCY_EURO,
```

Comment:
Dit is misschien een nieuwe story en niet van toepassing nu direct, maar ik vind niet dat een sdk ervan uit kan gaan dat alle gebruikers EURO als currency hebben, dat moet ergens variabel zijn of meegegeven worden.

Reply:
De api zal niet gaan werken met een andere currency vandaar dat wij in de plugins zeggen dat het altijd 'EUR' moet zijn